diff --git a/BUILD.bazel b/BUILD.bazel
index 9b48aee5..6ce86a4f 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -77,7 +77,7 @@ cc_library(
     ]),
     copts = select({
         ":windows": [],
-        "//conditions:default": ["-pthread"],
+        "//conditions:default": ["-pthread", "-Wno-unused-parameter"],
     }),
     defines = select({
         ":has_absl": ["GTEST_HAS_ABSL=1"],
diff --git a/googlemock/test/BUILD.bazel b/googlemock/test/BUILD.bazel
index da95ed58..2793cbca 100644
--- a/googlemock/test/BUILD.bazel
+++ b/googlemock/test/BUILD.bazel
@@ -46,6 +46,10 @@ cc_test(
         "//:windows": [],
         "//conditions:default": ["-pthread"],
     }),
+    copts = select({
+        "//:windows": [],
+        "//conditions:default": ["-Wno-unused-parameter"],
+    }),
     deps = ["//:gtest"],
 )
 
diff --git a/googletest/test/BUILD.bazel b/googletest/test/BUILD.bazel
index 156d5d4b..38e5520e 100644
--- a/googletest/test/BUILD.bazel
+++ b/googletest/test/BUILD.bazel
@@ -88,7 +88,7 @@ cc_test(
     ],
     linkopts = select({
         "//:windows": [],
-        "//conditions:default": ["-pthread"],
+        "//conditions:default": ["-pthread", "-lm"],
     }),
     deps = ["//:gtest_main"],
 )
